Source/WebCore/platform/graphics/GraphicsTypesGL.h

```diff
@@ -68,6 +68,38 @@ using GCGLContext = void*;
 typedef unsigned GLuint;
 #endif
 
+struct OwnedGLObject {
```

Comment:
I wonder if this would be more user-friendly if they were specific Owners for each type of GL object. Then they could call the respective `deleteFoo` method themselves.

Reply:
I'm glad you asked! I have a patch that does that. I wasn't exactly happy with it and started with this intermediate step.

> How did this get created without a bugzilla and radar? My guess: you started the work, already had a branch, and `git webkit` then skipped asking you to create one?

Exactly this. I added the bug ids and pushed but that doesn't appear to update the initial comment.